### PR TITLE
docker-compose-2

### PR DESCRIPTION
**Pull request by javatrap2020**
docker-compose-2

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,26 +1,69 @@
-version: "3.8"
+version: '3.7'
```
Comment (Forfend) [inline docker-compose.yml:1]: Any reasons to downgrade the version?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,26 +1,69 @@
...
+    container_name: "it-cluster-feedback-frontend"
     ports:
       - "4200:4200"
-    links:
-      - "db:feedback"
+    restart: always
     depends_on:
-      - "db"
\ No newline at end of file
+      - feedback-server
```
Comment (Forfend) [inline docker-compose.yml:37]: Why front-end depends on the back-end?

**Review by Forfend — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (javatrap2020) [reply docker-compose.yml:1]: Find a cause of errors to docker-compose. I  do change some versions.  Today change version 3.8, building  the docker-compose - good work. Then I  do change to 3.8 version.

File: docker-compose.yml (hunk 2 above)
Comment (javatrap2020) [reply docker-compose.yml:37]: I do front-end depends on the back-end for running to end.

Comment (Forfend) [reply docker-compose.yml:37]: Will it work without depends on back-end?

File: docker-compose.yml (hunk 1 above)
Comment (javatrap2020) [reply docker-compose.yml:1]: AWS-not-support 3-8. I checked AWS

Comment (javatrap2020) [reply docker-compose.yml:1]: 3-7 docker-compose OK

**Review by Forfend — APPROVED** [review]

**Pull request merged**